cmd/sfncli/runner.go

```diff
@@ -78,6 +80,10 @@ func (t *TaskRunner) Process(ctx context.Context, args []string, input string) e
 	if executionName != nil {
 		t.execCmd.Env = append(os.Environ(), "_EXECUTION_NAME="+*executionName)
 	}
+	if t.workDirectory != "" {
+		t.execCmd.Env = append(t.execCmd.Env, "WORK_DIR="+t.workDirectory)
+		defer os.RemoveAll(t.workDirectory)
```

Comment:
If sfncli is removing the directory, it seems like it should also create the directory? e.g. https://golang.org/pkg/io/ioutil/#TempDir

Reply:
https://github.com/Clever/sfncli/pull/9/commits/d5f77d08d488f03303accdcf13fbd39cd86d0dc2